packages/hardhat-chai-matchers/src/index.ts

```diff
@@ -0,0 +1 @@
+export { bnChai } from "./bnChai";
```

Comment:
I think we should add `chai.use(bnChai)` here

Reply:
Addressed in https://github.com/NomicFoundation/hardhat/pull/2428/commits/eb16ef2afdd2f0a1ab21e56aa1e4c62d115226da